generated/beats/fields.ecs.yml

```diff
@@ -847,6 +847,14 @@
       type: keyword
       ignore_above: 1024
       description: Unique container id.
+    - name: image.hash.all
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: 'An array of digests of the image the container was built on. Each
+        digest consists of the hash algorithm and value in this format: `algorithm:value`.'
```

Comment:
I would just add "Algorithm names in this value should align with the field names in the ECS hash field set" if that makes sense.